packages/protocol/contracts/governance/EpochRewards.sol

```diff
@@ -73,6 +73,7 @@ contract EpochRewards is
   event CarbonOffsettingFundSet(address indexed partner, uint256 fraction);
   event TargetValidatorEpochPaymentSet(uint256 payment);
   event TargetVotingYieldParametersSet(uint256 max, uint256 adjustmentFactor);
+  event TargetVotingYieldSet(uint256 target);
```

Comment:
We have a TargetVotingYieldUpdated event, suggest we stick with one event

Reply:
decided we may want to differentiate between the target being set via the conditions vs governance, so will still with the new event `TargetVotingYieldSet`